src/daipecore/widgets/Widget.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABC, abstractmethod
+import re
+
+
+class Widget(ABC):
+    @abstractmethod
+    def add_text(self, name: str, value: str, label: str = None):
+        pass
+
+    @abstractmethod
+    def add_select(self, name: str, value: list, label: str = None):
```

Comment:
když už jsme u toho tak dbutils to má taky pojmenovaný widgets: dbutils.widgets.dropdown()
jestli potenciálně lidi nejsou zvyklí na tenhle interface
např. Widgets.text(něco, ...) a Widgets.get(něco)